src/aosm/azext_aosm/tests/latest/recording_processors.py

```diff
@@ -0,0 +1,50 @@
+from azure.cli.testsdk.scenario_tests import (
```

Comment:
Can we have a heading on this file explaining what it is for please? I don't know what a Recording Processor is